### ISSUE TITLE
PS2 Switch Declaration Sniff doesn't allow use of return instead of break

### ISSUE THREAD

**Issue by PatrickRose** [squizlabs/PHP_CodeSniffer#559]
This is potentially me misreading PS2, but I think:

```php
switch ($expr) {
    case 0:
        echo 'First case, with a break';
        break;
    case 1:
        echo 'Second case, which falls through';
        // no break
    case 2:
    case 3:
         return 'This is allowed';
    case 4:
        echo 'Third case, return instead of break';
        return;
    default:
        echo 'Default case';
        break;
}
```

Should this also be allowed? The current sniff will error on case 3, since there's no break or comment explaining that fall through is allowed.

**Comment by PatrickRose** [comment]
I can't read. This is correct.
